test/scripts/apidoc/module.example.ts

```diff
@@ -9,3 +9,11 @@ export class ModuleFakerJsLinkTest {}
  * and [api docs](https://next.fakerjs.dev/api/).
  */
 export class ModuleNextFakerJsLinkTest {}
+
+/**
+ * This is a description for a module with a code example
+ *
+ * @example
+ * new ModuleFakerJsExampleTest()
+ */
+export class ModuleFakerJsExampleTest {}
```

Comment:
The `FakerJs` part refers to the target link, IMO there is no need to repeat that here.

```suggestion
export class ModuleExampleTest {}
```